person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = id
```

Comment:
- [ ] This `id` should be initialized to something since you are not providing it in the parameter.

  ```
  @id = Random.rand(1..100)
  ```